snuba/snuba_migrations/search_issues/0001_search_issues.py

```diff
@@ -0,0 +1,97 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Array,
+    Column,
+    DateTime,
+    IPv4,
+    IPv6,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("group_id", UInt(64)),
+    Column("search_title", String()),
+    Column("fingerprint", Array(String())),
```

Comment:
As far as I know I don't believe we have any queries that utilize fingerprint. We might have plans in the future though. If performance is a concern could we always materialize the array into a denormalized column?

Reply:
That begs a further question: how many fingerprints will there be in a given row? Having arbitrary sized arrays can lead to issues down the road.

Reply:
We won't enforce a limit but I don't see any reason a vertical team would be sending us a huge amount of fingerprints per issue. I think we can safely assume the size of the fingerprints array would be < 10 (or somewhere in that ballpark).

Reply:
Is there a limit being enforced somewhere? There should be a hard cap on this somewhere. If one doesn't already exist I would suggest adding something to your processor to cap the array. It can be a high limit, just something to prevent some crazy edge case from writing a huge array.

Reply:
I think it probably makes sense to limit this here just to be thorough and in our ingestion consumer during our validation step. Let's just put a limit of 100?

Reply:
Sounds good. Will introduce a hard limit and drop any fingerprints greater than 100.